book/website/community-handbook/coworking/coworking-daily.md

```diff
@@ -0,0 +1,58 @@
+(ch-coworking-daily)=
+# Daily Coworking Calls
+
+_The Turing Way_ team has been hosting coworking calls almost every day at 11:00 GMT/BST.
+These calls are similar to the Collaboration Cafes, but only 1-hour long.
+It is not scheduled on the days when there is a Collaboration Cafe already scheduled, the daily coworking call.
+
+Like Collaboration Cafe, we host these calls online using [Zoom](https://www.zoom.us/).
+We use the Shut up & Write! the method by providing allocated time, use two 25 minutes long Pomodoro sessions and take two short breaks to chat about our work and exchange what we got done.
+
+This call is hosted by the community member to allow people who are working on certain issues or PRs in _The Turing Way_ to come along and join the community members in their usual working hours.
+These calls will be also a good place to clarify doubts about the project, get their PRs reviewed, brainstorm on different ideas together, and stay connected with other members in the community.
+
+## Attending an online Collaboration Cafe
```

Comment:
```suggestion
## Attending an online coworking call
```

Swung by to review because @katoss was looking for the information and couldn't find it. Figured I'd stay and suggest a different heading which I think is just a typo. The link with the information is on line 25 but here it is to make it easier to click: [https://hackmd.io/@malvikasharan/TW-coworking](https://hackmd.io/@malvikasharan/TW-coworking

Reply:
That is indeed a typo. Thank you for correcting! 🌸